pxd.c

```diff
@@ -1118,6 +1125,25 @@ static void pxd_fuse_conn_release(struct fuse_conn *conn)
 {
 }
 
+static void pxd_wakeup(struct pxd_context *ctx)
+{
```

Comment:
It seems to me this newly introduced function pxd_wakeup from the timer, is running without any synchronization with the main IO path... so how would all the state variables used below be consistent?

Reply:
I think we don't have to add much synchronization for the timer since the timer is fired so frequently.  As you might have noticed, I am working on top of Vinod's changes.